api/SpeechRecognitionErrorEvent.json

```diff
@@ -80,6 +80,55 @@
           "deprecated": false
         }
       },
+      "SpeechRecognitionErrorEvent": {
+        "__compat": {
+          "description": "<code>SpeechRecognitionErrorEvent()</code> constructor",
+          "spec_url": "https://wicg.github.io/speech-api/#dom-speechrecognitionerrorevent-speechrecognitionerrorevent",
+          "support": {
+            "chrome": {
+              "version_added": false
```

Comment:
The oddly named `webkitSpeechRecognitionError` constructor is supported, can you take the results from http://mdn-bcd-collector.appspot.com/tests/api/webkitSpeechRecognitionError/webkitSpeechRecognitionError and put in here?

Note that the data for the interface itself is wrong, it suggests this event interface is no longer prefixed in Chrome, but it is. Can you fix that at the same time?